### PR TITLE
feat: chart with ingress

### PR DESCRIPTION
**Pull request by nmbarel**
Closes # 

## 📑 Description

add openshift route to the chart (controlled by values)

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Review by talboren — APPROVED** [review]
LGTM

File: chart/keep/values.yaml [hunk 1]
```diff
@@ -52,6 +52,12 @@ backend:
           - path: /
             pathType: ImplementationSpecific
     tls: []
+  route:
+    enabled: false
+    host: chart-example-backend.local
```
Comment (talboren) [inline chart/keep/values.yaml:57]: what's that exactly? why is it chart-example-backend and not keep-backend?

File: chart/keep/values.yaml [hunk 2]
```diff
@@ -115,6 +121,12 @@ frontend:
           - path: /
             pathType: ImplementationSpecific
     tls: []
+  route:
+    enabled: false
+    host: chart-example.local
```
Comment (talboren) [inline chart/keep/values.yaml:126]: > what's that exactly? why is it chart-example-backend and not keep-backend?

same

File: chart/keep/values.yaml (hunk 1 above)
Comment (nmbarel) [reply chart/keep/values.yaml:57]: taken from your ingress host value as to not deviate.

File: chart/keep/values.yaml (hunk 2 above)
Comment (nmbarel) [reply chart/keep/values.yaml:126]: same as above reply

**Review by talboren — APPROVED** [review]
LGTM!

**Pull request merged**